com.sap.cloud.lm.sl.cf.database/src/test/java/com/sap/cloud/lm/sl/cf/database/migration/executor/type/DatabaseTypeSetterFactoryTest.java

```diff
@@ -0,0 +1,90 @@
+package com.sap.cloud.lm.sl.cf.database.migration.executor.type;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DatabaseTypeSetterFactoryTest {
+
+    private static final String NON_EXISTING_DATABASE_TYPE_SETTER = "test";
+    private static final String BOOL_DEFAULT_DATABASE_TYPE_SETTER = "bool";
+    private static final String LONG_DEFAULT_DATABASE_TYPE_SETTER = "int8";
+    private static final String STRING_DEFAULT_DATABASE_TYPE_SETTER_1 = "varchar";
+    private static final String STRING_DEFAULT_DATABASE_TYPE_SETTER_2 = "text";
+
+    private DatabaseTypeSetterFactory databaseTypeSetterFactory;
```

Comment:
Remove this field. Use a local variable instead. Fields are only useful for sharing state between methods, but you're always re-initializing the value of this one.